release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2718,6 +2718,49 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+* Previously, an {product-title} cluster with a high number of internal services or user-managed load balancer IP addresses resulted in a delayed startup time for the OVN-Kubernetes service when the service attempted to install `iptables` rules on a node. With this release, the OVN-Kubernetes service can process a large number of services in a few seconds. Additionally, you can access a new log to view the status of installing `iptables` rules on a node. (link:https://issues.redhat.com/browse/OCPBUGS-32426[*OCPBUGS-32426*])
+
+* Previously, some container processes created by using the `exec` command persisted even when CRI-O stopped the container. Consequently, lingering processes led to tracking issues, causing process leaks and defunct statuses. With this release, CRI-O tracks the `exec` calls processed for a container and ensures that the processes created as part of the `exec` calls are terminated when the container is stopped. (link:https://issues.redhat.com/browse/OCPBUGS-32481[*OCPBUGS-32481*])
+
+* Previously, the *Topology* view in the {product-title} web console did not show the visual connector between a virtual machine (VM) node and other non-VM components. With this release, the visual connector is available so that you can view it to determine the interaction activity of a component. (link:https://issues.redhat.com/browse/OCPBUGS-32505[*OCPBUGS-32505*])
+
+* Previously, a logo in the masthead element of the {product-title} web console could grow beyond 60 pixels in height and this subsequently caused the masthead to increase in height. With this release, the masthead logo is constrained to a `max-height` of 60 pixels. (link:https://issues.redhat.com/browse/OCPBUGS-33058[*OCPBUGS-33058*])
+
+* Previously, when using the *Form* view in the {product-title} web console to remove an alternate service from a `Route` resource resulted in alternate service remaining in the cluster. With this release, after you delete an alternate service in this way, the alternate service is fully removed from the cluster.(link:https://issues.redhat.com/browse/OCPBUGS-33548[*OCPBUGS-33548*])
+
+* Previously, {product-title} cluster connections to the {azure-first} API were delayed because of an issue with the API's codebase. With this release, a timeout schedule is set any calls to the {azure-short} API, so that an API call that hangs for a period of time is terminated. (link:https://issues.redhat.com/browse/OCPBUGS-33127[*OCPBUGS-33127*])
```

Comment:
```suggestion
* Previously, {product-title} cluster connections to the {azure-first} API were delayed because of an issue with the API's codebase. With this release, a timeout schedule is set for any calls to the {azure-short} API, so that an API call that hangs for a period of time is terminated. (link:https://issues.redhat.com/browse/OCPBUGS-33127[*OCPBUGS-33127*])
```